ui/src/components/policy/PolicyList.js

```diff
@@ -95,13 +150,48 @@ export default class PolicyList extends React.Component {
             });
     }
 
+    onSubmitDuplicatePolicy() {
```

Comment:
This will be called when duplicating the policy ("versions" are not expanded). Duplicated version will have the same rules as the active version.